src/main/java/io/goobox/sync/common/overlay/OverlayIconProvider.java

```diff
@@ -19,7 +19,9 @@
 import java.nio.file.Path;
 
 public interface OverlayIconProvider {
-    
+
     public OverlayIcon getIcon(Path path);
 
+    public Path getIconFile(OverlayIcon state);
```

Comment:
Why is it necessary to bother the client (Storj/SIA sync app) providing the icon paths? I don't expect any difference in the overlay icons for both Storj and SIA sync apps. Can't the common project provide this info on its own?

Reply:
I just want to avoid hardcoding vendor specific file paths. Since sycn-storj and sync-sia have different binaries for each platforms, I thought it's better they are responsible for maintain such paths. (I think the GUI is responsible for specifying paths because it decides file structures, though)

Of course, we can specify icon paths in `OverlayHelper` but the paths are little bit strange and out of repository such as `../../../icons/ok.icns`, etc.

Reply:
Yes, but it would be the same issue for storj/sia-sync apps. Perhaps, we should unify the package structure for both sync apps in a way that it remains consistent when both are installed by the Goobox installer. Then we can assume that the icons can be always found at `<working-dir>/overlay-icons` or something like that.

Unifying the package structure would also help reducing the installation size as both sync apps share some common libraries.

Reply:
Another, simpler, approach would be to use an environment variable for the location of these icons. If the env var is not set then assume they are under the app's working directory.

Reply:
or system property instead of env var.

Reply:
I agree that storj/sia-sync apps should have a similar file structure and share libraries. 

For now, using system property to specify icon file paths is good for me.